actix/src/sec-5-arbiter.md

```diff
@@ -15,89 +15,148 @@ for your actors to run inside. When you call `start()` on your actor it is then
 running inside of the System Arbiter's thread. In many cases, this is all you
 will need for a program using Actix.
 
+While it only uses one thread, it uses the very efficient event loop pattern
+which works well for asynchronous events. To handle synchronous tasks that
+require heavy CPU power, it's better to avoid blocking the event loop and
+instead offload the computation to other threads. For this usecase, read the
+next section and consider using [`SyncArbiter`](./sec-6-sync-arbiter.md).
+
+## The event loop
+
+> If you're familiar with event loops in other paradigms, such as libuv (upon
+> which Node runs) or tokio (upon which Actix is implemented), each Arbiter has
+> a similar processing paradigm.
+
+> Specifically, if you're familiar with how JavaScript handles asynchronicity
+> and how the single-threaded model makes it so JS mostly doesn't need
+> synchronization primitives, this should feel very familiar to you.
+
+One `Arbiter` is in control of one thread with one task queue, and when tasks
+get spawned in an Actor's context or using `Arbiter::spawn`, the task gets
+queued up for execution in that thread.
+
+If you pay close attention to way both `ActorFuture` and the methods in the
```

Comment:
I think this first part about "close attention" is condescending and not required. It would be better to just say.

"As the `Actor` trait references it's context with &mut self and &mut Self::Context, this means that the Actor must have exclusive access to itself. There can only be a single instance of this Actor executing at any time. This guarantees that there can not be a concurrent execution of the same Actor, removing the requirement for synchronisation primitives."

Reply:
I'm gonna try something else here. Let me know what you think:

"Because Arbiters run futures on only one event loop on only one thread, Actors can spawn futures using the Arbiter that mutably reference the Actor. This property allows Actors to specify asynchronous behavior to access Actor state without using synchronization primitives. For more, see `ActorFuture` <with a link to ActorFuture>"

Reply:
This seems okay to me, but still a bit dense. But not enough for me to hold up a review any longer.